package.json

```diff
@@ -6,33 +6,34 @@
     "enzyme-to-json": "^3.3.4",
     "eslint-plugin-react-native": "^3.2.1",
     "exp": "^55.0.5",
-    "expo": "^40.0.0",
-    "expo-status-bar": "~1.0.3",
+    "expo": "^41.0.0",
+    "expo-status-bar": "~1.0.4",
     "moment": "^2.22.2",
     "npm": "^6.14.6",
     "react": "16.13.1",
     "react-geocode": "^0.1.2",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.0.tar.gz",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz",
     "react-native-elements": "^3.3.2",
-    "react-native-gesture-handler": "~1.8.0",
+    "react-native-gesture-handler": "~1.10.2",
     "react-native-keyboard-aware-scroll-view": "^0.9.2",
     "react-native-maps": "0.27.1",
     "react-native-open-maps": "0.3.5",
-    "react-native-reanimated": "~1.13.0",
-    "react-native-safe-area-context": "3.1.9",
-    "react-native-screens": "~2.15.0",
-    "react-native-svg": "^12.1.0",
-    "react-native-webview": "11.0.0",
-    "react-navigation": "4.0.10",
-    "react-navigation-drawer": "^2.2.2",
-    "react-navigation-stack": "^1.9.3",
-    "react-navigation-tabs": "^2.5.5",
+    "react-native-reanimated": "~2.1.0",
```

Comment:
@RyanTG, was there a reason to bump react-native-reanimated? We are getting the interpolate vs interpolateNode error due to the react navigation. From what I can tell we need to use react-navigation 5+ to be able to use react-native-reanimated 2+ (at least in order to avoid the warning). When I change this back to `1.13.0` the interpolate warning goes away. Everything _seems_ with react navigation as well, so I'm thinking we go back to something 1.X.X until we eventually get to moving to react navigation 5.

Reply:
Ah sweet. That package upgrade happened automatically with `expo upgrade`.

nice find. I could not figure out where we were using that reanimated package.